Doc/library/enum.rst

```diff
@@ -925,3 +925,17 @@ Notes
        >>> from enum import IntEnum
        >>> class MyIntEnum(IntEnum):
        ...     __str__ = IntEnum.__str__
+
+   Additionally, it's possible to end up shadowing a built-in function from
+   a derived baseclass such as :meth:`str.title` or :meth:`int.bit_count`,
+   which can cause unexpected issues at runtime.
+
+   Consider the following example::
+
+       >>> from enum import StrEnum
+       >>> class MyStrEnum(StrEnum):
+       ...     title = "RED"
+       >>> MyStrEnum.title.title()
```

Comment:
I think it is worth showing that other `str` methods work fine: `.uppercase()` for example.

Reply:
Not needed.  `.uppercase` will stop working as soon as an `uppercase` member is created -- which is the point of the doc addition.